tutorials/remote-staking-xcm.md

```diff
@@ -179,6 +242,47 @@ Verify that the structure of your XCM message resembles the below image, then pr
+
+```javascript
+// Import
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+// Construct API provider
+const wsProvider = new WsProvider('wss://frag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network');
+const api = await ApiPromise.create({ provider: wsProvider });
+
+// Import the keyring as required
+import { Keyring } from '@polkadot/api';
+
+// Initialize wallet key pairs
+const keyring = new Keyring({ type: 'sr25519' });
+const otherPair = await keyring.addFromUri('YOUR DEV SEED PHRASE HERE');
+console.log(`Derived Address from Private Key: ${otherPair.address}`);
+
+// Form the transaction
+//const tx = await api.tx.balances.transfer(bob, 1000000000000);
+const dest = { V1: { parents: 0, interior: { X1: { Parachain: 1000 } } } };
+const message = { V2: [{WithdrawAsset:[{id:{concrete:{parents:0,interior:{X1:{PalletInstance: 3}}}},fun: {Fungible: 100000000000000000n}}]},{BuyExecution:[{id:{Concrete:{parents:0,interior:{X1:{PalletInstance: 3}}}},fun: {Fungible: 100000000000000000n}}, {unlimited:null}]},{Transact:{originType: "SovereignAccount",requireWeightAtMost:40000000000n,call:{encoded:'0x0c113a7d3048f3cb0391bb44b518e5729f07bcc7a45d000064a7b3b6e00d00000000000000002c01000025000000'}}}]};
+let tx = api.tx.xcmPallet.send(dest, message);
+
+// Retrieve the encoded calldata of the transaction
+const encodedCallData = tx.toHex();
+console.log("Encoded call data is" + encodedCallData);
+
+// Sign and send the transaction
+const txHash = await tx.signAndSend(otherPair);
+
+// Show the transaction hash
+console.log(`Submitted with hash ${txHash}`);
+```
+
+!!! note
+    Remember that your multilocation derivative account must be funded with at least 1.1 DEV or more to ensure you have enough to cover the stake amount and transaction fees.
 
-And that’s it! To verify that your delegation was successful, you can visit [Subscan](https://moonbase.subscan.io/){target=_blank} to check your staking balance. Be advised that it may take a few minutes before your staking balance is visible on Subscan. Additionally, be aware that you will not be able to see this staking operation on Moonscan, because we initiated the delegation action directly via the parachain staking pallet (on the substrate side) rather than through the staking precompile (on the EVM). 
+In the above snippet, besides submitting the remote staking via XCM transaction, we also print out the encoded call data and the txn hash to assist with any debugging. And that’s it! To verify that your delegation was successful, you can visit [Subscan](https://moonbase.subscan.io/){target=_blank} to check your staking balance. Be advised that it may take a few minutes before your staking balance is visible on Subscan. Additionally, be aware that you will not be able to see this staking operation on Moonscan, because we initiated the delegation action directly via the parachain staking pallet (on the substrate side) rather than through the staking precompile (on the EVM). 
```

Comment:
```suggestion
In the above snippet, besides submitting the remote staking via XCM transaction, we also print out the encoded call data and the transaction hash to assist with any debugging. And that’s it! To verify that your delegation was successful, you can visit [Subscan](https://moonbase.subscan.io/){target=_blank} to check your staking balance. Be advised that it may take a few minutes before your staking balance is visible on Subscan. Additionally, be aware that you will not be able to see this staking operation on Moonscan, because we initiated the delegation action directly via the parachain staking pallet (on the Substrate side) rather than through the staking precompile (on the EVM). 
```